### PR TITLE
add more info about who can access orchestrate and how

### LINKED ISSUE #10: Clearly indicate that Docker registry access is granted only to Partners
Explain that Orchestrate Docker repos access is granted to partners only.

Later, maybe find a way to automatically grant access/generate credentials to users within partner orgs.

### PR DESCRIPTION
**Pull request by NicolasMassart**
Clearly indicate that Docker registry access is granted only to Partners and customers

fixes #10

### CODE REVIEW

**Review by julien-marchand — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -7,6 +7,19 @@
 PegaSys Orchestrate is a platform that enables enterprises to easily build secure
 and reliable applications on Ethereum blockchains.
 
+> **Important:** Pegasys Orchestrate Quickstart is only available for Orchestrate customers and
+> PegaSys partners.
+>
+> [Contact PegaSys](https://pegasys.tech/contact/) if you want to purchase Orchestrate or are interested in our partner program.
+>
+> If you are already a partner or customer but we did not provide the credentials to access the Docker registry, request access on [Pegasys Zendesk support site](http://pegasys.zendesk.com/)
+> (create an account with your company email address or login with an existing one) or by sending an email from your work email to
```
Comment (MadelineMurray) [inline README.md]: I'm not sure why the _create an account...._ part is in brackets?  Would this be clearer as: 

To access Zendesk, create an account with your company email or login using an existing Zendesk account. Alternatively, send an email from your company email to support@pegasys.tech

File: README.md [hunk 2]
```diff
@@ -7,6 +7,19 @@
...
+> [support@pegasys.tech](mailto:support@pegasys.tech).
+>
+> You can use this Quickstart only if you received these credentials from Pegasys to connect to the registry.
```
Comment (MadelineMurray) [inline README.md]: if you received -> if you have received

File: README.md [hunk 3]
```diff
@@ -7,6 +7,19 @@
...
+>
+> **YOU CANNOT CONTINUE WITHOUT THE REGISTRY CREDENTIALS.**
```
Comment (MadelineMurray) [inline README.md]: I think having this in bold without the caps if sufficient emphasis.

**Review by MadelineMurray — APPROVED** [review]
Minor comments

**Pull request merged**